tests/mocks/MockDidCache.ts

```diff
@@ -1,13 +0,0 @@
-import Transaction from '../../src/Transaction';
```

Comment:
Can we please bring back this file? My upcoming unit test still depends on the mock DidCache.

Reply:
Done - but since the DidCache interface has new methods I had to extend the class a bit.